src/pages/questions/ProductInformation.jsx

```diff
@@ -42,11 +51,53 @@ const ProductInformation = ({ question }) => {
   const { t } = useTranslation();
   const [productData, setProductData] = React.useState({});
   const [hideImages, setHideImages] = React.useState(getHideImages);
+  const [flagged, setFlagged] = React.useState([]);
+
+  const flagImage = (src, barcode) => {
+    const imgid = getImageId(src);
+    axios
+      .put(
+        `https://amathjourney.com/api/off-annotation/flag-image/${barcode}`,
+        {
+          mode: "no-cors",
+          imgid: imgid,
+          url: src,
+        }
+      )
+      .catch(() => {
+        console.log("Image flagged");
+      });
+    const newFlagged = [...flagged];
+    newFlagged.push(imgid);
+    setFlagged(newFlagged);
+  };
+
+  const deleteFlagImage = (src, barcode) => {
+    const imgid = getImageId(src);
+    axios
+      .delete(
+        `https://amathjourney.com/api/off-annotation/flag-image/${barcode}`,
+        {
+          mode: "no-cors",
+          data: {
+            imgid: imgid,
+          },
+        }
+      )
+      .catch(() => {
+        console.log("Something went wrong. Image could not be flagged");
```

Comment:
nit: no need for the `console.log` as it is not useful for the client, you could leave `.catch(() => {})` and it will be fine